<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeriodsUI</class>
 <widget class="QWidget" name="PeriodsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="peLabel">
     <property name="text">
      <string>Perioden</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="peSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="cadLabel">
     <property name="text">
      <string>Perioden instellen op de cyclus lengte</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="cadCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="AnimatedSlideWidget2" name="cadWidget" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="taLabel">
        <property name="text">
         <string>Ook bij aanwezigheid van een trend</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="taCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="micpLabel">
        <property name="text">
         <string>Minimum perioden</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="micpSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="macpLabel">
        <property name="text">
         <string>Maximum perioden</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="macpSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="caLabel">
        <property name="text">
         <string>Versterkings factor</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="caSpinBox">
        <property name="suffix">
         <string> x</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>taLabel</zorder>
     <zorder>taCheckBox</zorder>
     <zorder>micpLabel</zorder>
     <zorder>micpSpinBox</zorder>
     <zorder>macpLabel</zorder>
     <zorder>macpSpinBox</zorder>
     <zorder>caLabel</zorder>
     <zorder>caSpinBox</zorder>
     <zorder>taLabel</zorder>
     <zorder>taCheckBox</zorder>
     <zorder>micpLabel</zorder>
     <zorder>micpSpinBox</zorder>
     <zorder>macpLabel</zorder>
     <zorder>macpSpinBox</zorder>
     <zorder>caLabel</zorder>
     <zorder>caSpinBox</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedSlideWidget2</class>
   <extends>QWidget</extends>
   <header>animatedslidewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
